torch/csrc/jit/script/concrete_module_type.cpp

```diff
@@ -210,15 +224,6 @@ void ConcreteModuleTypeBuilder::addFailedAttribute(
   failedAttributes_.emplace(std::move(name), std::move(failureReason));
 }
 
-c10::optional<py::object> ConcreteModuleType::findConstant(
```

Comment:
This method implementation was removed but it's still in the header file, please remove there as well.